source/_integrations/jellyfin.markdown

```diff
@@ -0,0 +1,40 @@
+---
+title: Jellyfin
+description: Instructions on how to integrate the Jellyfin integration into Home Assistant.
+ha_category:
+  - Media Source
+ha_release: '2021.1'
+ha_iot_class: Local Polling
+ha_codeowners:
+  - '@j-stienstra'
+ha_config_flow: true
+ha_domain: jellyfin
+---
+
+The `jellyfin` integration exposes a [Jellyfin](https://jellyfin.org/) server as a Media Source in Home Assistant.
+Support is currently limited to music libraries only. Other libraries will not appear in the Media Browser. This integration has been tested with Jellyfin server version 10.6.4, but should support older versions as well.
+
+## Configuration
+
+The Jellyfin integration is configured through the Home Assistant user interface. In the Configuration page, select Integrations and click on the `+` button and choose Jellyfin.
+
+{% configuration %}
```

Comment:
This configuration block is used for YAML configuration. If you want to supply such an index, please use the basic configuration block instead (examples can be found in the Lovelace card documentation).

Reply:
Done